crates/pallet-domains/src/staking.rs

```diff
@@ -191,8 +191,7 @@ pub struct Operator<Balance, Share, DomainBlockNumber> {
     /// The status of the operator, it may be stale due to the `OperatorStatus::PendingSlash` is
     /// not assigned to this field directly, thus MUST use the `status()` method to query the status
     /// instead.
-    /// TODO: update the filed to `_status` to avoid accidental access in next network reset
-    status: OperatorStatus<DomainBlockNumber>,
+    _status: OperatorStatus<DomainBlockNumber>,
```

Comment:
Maybe the name should be something else entirely? `_` has the purpose of not triggering unused field lints. Not sure why it needs to be renamed if it is hidden in the first place.

Reply:
I was trying to emphasize this field should not be queried directly. Yeah, perhaps the comment and private field would be enough, I will rename it back to `status` or it would be great if you have other suggestions.

Reply:
I was thinking something like `partial_status` or `preliminary_status` or something similar, what makes sense in this context

Reply:
Thanks! renamed to `partial_status.`